api/v1beta2/foundationdbcluster_types.go

```diff
@@ -219,6 +223,15 @@ type FoundationDBClusterSpec struct {
 	UseUnifiedImage *bool `json:"useUnifiedImage,omitempty"`
 }
 
+// ProcessesConfig represents the configuration for a processes. This will be used as a template for all processes
+// of with the specified process class.
+type ProcessesConfig struct {
+	// Class the process class that is affected by this configuration.
+	Class ProcessClass `json:"class,omitempty"`
```

Comment:
I think this is going to introduce a new string field in the CRD with no max-length. Maybe we should take this opportunity to put a max-length annotation on the ProcessClass type.

Reply:
Good point 👍